hw/riscv/ot_earlgrey.c

```diff
@@ -360,6 +380,14 @@ static const IbexDeviceDef ot_eg_soc_devices[] = {
             IBEX_DEV_UINT_PROP("idcode", EG_RV_DM_TAP_IDCODE)
         ),
     },
+    [OT_EG_SOC_DEV_LC_CTRL_TAP_CTRL] = {
+        .type = TYPE_TAP_CTRL_RBB,
+        .cfg = &ot_eg_soc_lc_ctrl_tap_ctrl_configure,
+        .prop = IBEXDEVICEPROPDEFS(
+            IBEX_DEV_UINT_PROP("ir_length", IBEX_TAP_IR_LENGTH),
+            IBEX_DEV_UINT_PROP("idcode", EG_RV_DM_TAP_IDCODE)
```

Comment:
```suggestion
            IBEX_DEV_UINT_PROP("idcode", EG_LC_CTRL_TAP_IDCODE)
```
? And also, do we need to consider the `EG_COMBINED_TAP_IDCODE` somewhere? I am not so clear about the specifics with the combined TAP.

It seems from the docs like in reality we should have one TAP interface which is multiplexed based upon the values of the TAP straps at startup?

Reply:
> And also, do we need to consider the EG_COMBINED_TAP_IDCODE somewhere? I am not so clear about the specifics with the combined TAP.

The combined TAP is only for DJ. EG has separate TAPs.